cli/cli/infra.py

```diff
@@ -64,10 +65,11 @@
 class InfraManager(object):
     def __init__(self):
         # Initialise SCW client
-        self.scw_client = Client.from_config_file_and_env(profile_name="dev")
+        self.scw_client = Client.from_config_file_and_env()
```

Comment:
We could also add a CLI flag for this

Reply:
Yes that would be nice, would need to add it to all the commands that use the API which would be a lot of changes so won't include in this PR.